frontend/appflowy_flutter/lib/startup/startup.dart

```diff
@@ -136,6 +138,7 @@ class FlowyRunner {
           if (isAppFlowyCloudEnabled) InitAppFlowyCloudTask(),
           const InitAppWidgetTask(),
           const InitPlatformServiceTask(),
+          getIt<CachedRecentService>(),
```

Comment:
should not use it here. Create a launch task, wrap it, and dispose of the sources if needed.